docs/development/README.md

```diff
@@ -77,13 +78,13 @@ unless the CHANGELOG is updated._
 ### View Available Commands
 
 ```bash
-$ make
+make
```

Comment:
So this is one of the things that my "auto format on save" does in vscode for`md` files. Hmm I'll go fix these up and just turn change my vscode stuff.

Reply:
We should use `make` instead of `$ make`. I never had the chance to properly implement #257 yet so everyone's VSCode settings in this repo are identical.

Reply:
I just went and reverse all the formatting stuff because its annoying me, also makes it easier for me to review my own diff! I commit the ones you commented but may as well do them all (trailing spaces, lines, md format, tabs, issues/pr links, etc.) together at some point.